.travis.yml

```diff
@@ -3,12 +3,14 @@ matrix:
     include:
         - dist: bionic
         - dist: focal
+        - dist: jammy

```

Comment:
is this file in use anywhere ?

Reply:
@SamerKhshiboun, I am not aware of any usage of it in the repo. May be, we can check if it's used in the dailies/weeklies? If not, we can remove this.